astropy/cosmology/flrw.py

```diff
@@ -1485,6 +1454,99 @@ def arcsec_per_kpc_proper(self, z):
         return u.arcsec / (self.angular_diameter_distance(z).to(u.kpc) *
                            arcsec_in_radians)
 
+    # -----------------------------------------------------
+
+    def __equiv__(self, other):
+        """FLRW equivalence. Use ``.is_equivalent()`` for actual check!
+
+        Parameters
+        ----------
+        other : `~astropy.cosmology.FLRW` subclass instance
+            The object in which to compare.
+
+        Returns
+        -------
+        `True` or `NotImplemented`
```

Comment:
I think it should be allowed to return `False` if that is obviously correct (same class and different parameters)

Reply:
Agreed. I have a local implementation and it works quite nicely. Saves doing a check from ``other``.

Reply:
Done!
``__equiv__`` can return ``False``, can save time in ``is_equivalent``.